src/parser/bison_parser.y

```diff
@@ -557,10 +566,14 @@ column_type:
 	|	INTEGER { $$ = ColumnType{DataType::INT}; }
 	|	LONG { $$ = ColumnType{DataType::LONG}; }
 	|	FLOAT { $$ = ColumnType{DataType::FLOAT}; }
+	|   DECIMAL '(' INTVAL ',' INTVAL ')' { $$ = ColumnType{DataType::FLOAT}; }
 	|	DOUBLE { $$ = ColumnType{DataType::DOUBLE}; }
+	|	REAL { $$ = ColumnType{DataType::FLOAT}; }
 	|	VARCHAR '(' INTVAL ')' { $$ = ColumnType{DataType::VARCHAR, $3}; }
+	|	CHARACTER VARYING'(' INTVAL ')' { $$ = ColumnType{DataType::VARCHAR, $4}; }
 	|	CHAR '(' INTVAL ')' { $$ = ColumnType{DataType::CHAR, $3}; }
 	|	TEXT { $$ = ColumnType{DataType::TEXT}; }
+	|   TIME { $$ = ColumnType{DataType::TIME}; }
```

Comment:
see comment to decimal above

Reply:
see answer above.

Reply:
It is now introduced in https://github.com/hyrise/sql-parser/pull/177/commits/83d45ac410bdf109e981f1a4b8bb596a409aa13c